### PR TITLE
core(network-request): use ms instead of seconds

### PR DESCRIPTION
**Pull request by connorjclark**
split out from #14311

Changes the network request (aka network records) start and end time properties to be _in ms_, instead of seconds. This should not impact any audit values - `network-requests` was already converted to ms, and critical-request-chain timings aren't impacted as I retained them to be in seconds.

### CODE REVIEW

File: core/lib/dependency-graph/network-node.js [hunk 1]
```diff
@@ -25,14 +25,14 @@ class NetworkNode extends BaseNode {
    * @return {number}
    */
   get startTime() {
-    return this._record.startTime * 1000 * 1000;
+    return this._record.startTime * 1000;
```
Comment (paulirish) [inline core/lib/dependency-graph/network-node.js:28]: seems kinda awkward that the lantern depgraph and crc are still in seconds. 

wdyt about changing both of those in here, too?

**Review by paulirish — APPROVED** [review]
nice! 
so happy to be so much closer to seconds-free.

one Q below that's got me pensive.

File: core/lib/dependency-graph/network-node.js (hunk 1 above)
Comment (connorjclark) [reply core/lib/dependency-graph/network-node.js:28]: I mentioned this in the other PR, but I would like to keep lantern changes in a separate PR. I expect it to change a number of things in a trivial way (snapshots, accuracy) due to the limits of floating point precision. It should be a quick change I can follow up with when this lands.

Comment (paulirish) [reply core/lib/dependency-graph/network-node.js:28]: maybe add some comments in network-node/base-node/cpu-node and crc that these times are in seconds?
and/or a TODO?

cuz right now its kinda impossible to tell from the code.

**Pull request merged**